src/test/java/uk/gov/justice/services/test/domain/EventsCheckerTest.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.justice.services.test.domain;
+
+import static uk.gov.justice.services.test.DomainTest.eventsFromFileNames;
+import static uk.gov.justice.services.test.DomainTest.jsonNodesFrom;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+
+public class EventsCheckerTest {
```

Comment:
I think we need some further unit tests for this. Currently, we have
Pass: Expected ["something happened"] Actual ["something happened"]
Fail: Expected ["something happened"] Actual []
Fail: Expected ["something happened"] Actual ["something-else-happened"]

We should also cover off:
Pass: Expected [] Actual []
Pass: Expected ["something happened", "something-else-happened"] Actual ["something happened", "something-else-happened"]
Fail: Expected [] Actual ["something happened"]
Fail: Expected ["something happened"] Actual ["something-else-happened"]
Pass: Expected ["something happened", "something-else-happened"] Actual ["something happened"]
Fail: Expected ["something happened", "something-else-happened"] Actual ["something-else-happened", "something-else-happened"]